src/condvar.rs

```diff
@@ -360,6 +379,7 @@ impl Condvar {
             }
 
             // ... and re-lock it once we are done sleeping
+            // REVIEW: how is it possible that TOKEN_HANDOFF gets produced here?
```

Comment:
It is produced from `RawMutex::unlock_slow`.

Reply:
Can that be documented here and/or somewhere? Knowing that a head of time would have made it much easier to read...